Buscorepuestos/buscorepuestos-web
# Feature/br 456 conectar algolia

## What type of PR is this? (check all applicable)


## Description
Integrar Algolia para la visualización y búsqueda de productos.

## Related Tickets & Documents
https://linear.app/buscorepuestos/issue/BR-456/conectar-algolia

## QA Instructions, Screenshots, Recordings
Acceder a [http://localhost:3000/store](url)
Verificar que se muestran los productos de Algolia.
Probar el buscador y verificar que realiza correctamente la búsqueda.


Also make sure you check this browser compatibility version for desktop and mobile:

## Checklist

DO NOT send the PR without completing the checklist above:


## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

- [x] No, and this is why: No se hicieron cambios críticos a ningún componente.

## Are there any post deployment tasks we need to perform?
